### PR TITLE
adding helm linting and install checks

### PR DESCRIPTION
**Pull request by fredericfran-gds**
This PR adds to the GitHub Actions CI checks:
1. Helm Linting
2. Helm Install checks

The checks are performed by the Helm tool
[ct](https://github.com/helm/chart-testing)

**Note**
ct works by comparing a commit/PR against the
`main` branch (configurable via ct.yml file).

Ref:
1. [trello card](https://trello.com/c/N8aXJvkK/624-configure-github-actions-for-helm)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -5,12 +5,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```
Comment (sengi) [inline .github/workflows/ci.yml:9]: Good idea!

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -5,12 +5,27 @@ jobs:
...
       - uses: ruby/setup-ruby@v1
```
Comment (sengi) [inline .github/workflows/ci.yml:10]: Slightly unrelated but it doesn't look like we're using Ruby in this workflow - or am I missing something?

**Review by sengi — APPROVED** [review]
Cool thanks for adding this.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (fredericfran-gds) [reply .github/workflows/ci.yml:10]: yeah, I guess it is to do with the Signon secret generation code, it runs some rspecs for it.

Comment (sengi) [reply .github/workflows/ci.yml:10]: Yeah I guessed the same, just couldn't find where :/

**Pull request merged**